client/components/mma/switch/SwitchComplete.tsx

```diff
@@ -75,6 +76,14 @@ export const SwitchComplete = () => {
 			{!switchContext.isFromApp && (
 				<section css={sectionSpacing}>
 					<SignInBanner />
+					<LinkButton
+						css={css`
```

Comment:
This should be `cssOverrides` rather than `css`?

In general it's good to avoid adding style overrides to components if you can though. (Sometimes it _is_ the only way to do it though.)

It might be nicer to wrap the banner and button components with a `<Stack>` which will apply spacing between the components automatically:

```jsx
<Stack space={6}>
    <SignInBanner />
    <div>
        <LinkButton ...>
    </div>
</Stack>
```

(I've added a `<div>` around the button as `<Stack>` makes the width of all of the child elements 100% which you probably don't want for the button.)

Reply:
Ahh ok! Nice suggestions - I've changed it. I'll just chuck it on to CODE and double check it's working as expected 👀

Reply:
Tested successful in CODE